flake.nix

```diff
@@ -27,23 +27,23 @@
             ocamlPackages = ocamlPackages_dev;
           });
         in {
+          formatter = pkgs.callPackage ./nix/formatter.nix { };
           devShells = {
             default = (pkgs.mkShell {
               inputsFrom = [ pollinate ];
+
               buildInputs = with pkgs;
                 with ocamlPackages_dev; [
                   ocaml-lsp
-                  ocamlformat
+                  ocamlformat_0_20_1
                   odoc
                   ocaml
                   dune_3
-                  nixfmt
+                  treefmt
```

Comment:
You probably still want `nixfmt` since `treefmt` will call out to it

Reply:
My bad, done.